src/components/UserCreatedBounties/UserBountyPage/BountyPost/BountyPostReactions/BountyCommentsContainer.tsx

```
@@ -0,0 +1,388 @@
+const BountyCommentsContainer = ({ className, id, comments }: { id: number | null | undefined; className: string; comments: { [index: string]: IComment[] } }) => {
+	const [comment, setComments] = useState<{ [index: string]: IComment[] }>(comments);
+	const [openLoginModal, setOpenLoginModal] = useState<boolean>(false);
+	const [showOverallSentiment, setShowOverallSentiment] = useState<boolean>(true);
+	const [sentimentsPercentage, setSentimentsPercentage] = useState<ISentimentsPercentage>({ against: 0, for: 0, neutral: 0, slightlyAgainst: 0, slightlyFor: 0 });
+	const [loading, setLoading] = useState(true);
+	const { network } = useNetworkSelector();
+	const [filterSentiments, setFilterSentiments] = useState<ESentiments | null>(null);
+	const router = useRouter();
+	let allComments = Object.values(comments)?.flat() || [];
+	const { resolvedTheme: theme } = useTheme();
+	const [reasonForNoComment, setReasonForNoComment] = useState<String | null>(null);
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)